packages/devtools/lib/src/ui/viewport_canvas.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:html' hide VoidCallback;
 import 'dart:js';
 
+import 'package:devtools/src/ui/ui_utils.dart';
```

Comment:
prefer a relative import inside of lib/?

We may also want to follow this PR up with a check on the CI to fail if there's a package:devtools import from within the lib dir.

Reply:
yeah that check would be nice. I keep forgetting to fix up the result of the auto-import.